.travis.yml

```diff
@@ -5,25 +5,52 @@ cache:
     - $HOME/.composer/cache
 
 php:
-    - 5.5
-    - 5.6
-    - 7.0
+    - "5.5"
+    - "5.6"
+    - "7.0"
+    - "7.1"
+    - nightly
     - hhvm
 
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-          env: SYMFONY_VERSION=2.8.*
-        - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
+        - php: "5.5"
+          env:
+              - "SYMFONY_VERSION=~2.8.0"
+              - "COMPOSER_FLAGS=--prefer-lowest"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.0.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.1.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.2.0@dev"
+    allow_failures:
+        - php: nightly
```

Comment:
I think that's important but if the tests don't fail no one will ever notice the library is not compatible with the next version of php.
I would personnally keep this build but threat it like the others.